ovos_m2v_pipeline/__init__.py

```
@@ -14,236 +11,439 @@
+    def build(
+        cls,
+        model,
+        sentences: List[str],
+        labels: List[str],
+        k: int = 5,
+        random_state: int = 42,
+    ) -> "PrototypeIntentStore":
+        """Build a store from parallel *sentences* / *labels* lists."""
+        store = cls()
+        label_to_sentences: Dict[str, List[str]] = {}
+        for sent, lbl in zip(sentences, labels):
```

Ruff (0.15.2)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)